bedrock/firefox/templates/firefox/channel/base.html

```diff
@@ -46,18 +46,20 @@ <h2 class="c-sub-navigation-title">
 
   {% block channels %}{% endblock %}
 
-  {% call picto_card(
-    title=ftl('firefox-channel-see-something-that-isnt-working'),
+  <aside class='mzp-t-picto-center'>
```

Comment:
nitpick: should use double quotes for attributes:
```suggestion
  <aside class="mzp-t-picto-center">
```

Reply:
d'oh -- thanks for noticing.